aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -64,12 +64,16 @@ Update the `SeedData` class so that it provides a value for the new column. A sa
 
 [!code-csharp[Main](start-mvc/sample/MvcMovie/Models/SeedDataRating.cs?name=snippet1&highlight=6)]
 
-Build the solution then open a command prompt. Enter the following commands:
+Build the solution then open a command prompt in the solution folder. Enter the following commands:
 
 ```console
 dotnet ef migrations add Rating
 dotnet ef database update
 ```
+Note: If you get the error message `No executable found matching command "dotnet-ef"`:
```

Comment:
There are a lot of causes for that error, see my Migrations tutorial where I link to Julie Lerman's blog for troubleshooting info.